### DISCUSSION TITLE
Locked Version dependencies?

### DISCUSSION THREAD

**Discussion by Lennix** [renovatebot/renovate#21936] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I use a packageRule to lock symfony to 5.x

`{
      "matchPackagePatterns": ["^symfony/"],
      "groupName": "symfony",
      "allowedVersions": "<6.0.0"
    }`

I was wondering if renovate is smart enough to update the remaining dependencies only if they are compatible with my locked version of symfony or if this is configurable somehow?

### What you tried so far.

Setup renovate.json and checked expected PRs

**Comment by rarkins** [comment, marked as answer]
Not possible or configurable
